bes.spec.all_static

```diff
@@ -201,6 +201,7 @@ exit 0
 %{_bindir}/get_dmrpp
 %{_bindir}/ingest_filesystem
 %{_bindir}/ingest_s3bucket
+%{_bindir}/retriever
```

Comment:
If you make the change I suggest further along, I think this can be removed (and the similar line in bes.spec)

Reply:
Done